prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.html

```diff
@@ -55,32 +57,45 @@
 
         <div class="col-sm-12 pl-5">
 
-          <div class="row">
-            <div class="col">
+          <mat-form-field class="w-100">
+            <input matInput
+                   placeholder="Licence Number from College"
+                   formControlName="licenseNumber">
+            <mat-error *ngIf="form.get('licenseNumber').hasError('required')">Required</mat-error>
+            <mat-error *ngIf="form.get('licenseNumber').hasError('alphanumeric')">
+              Must contain only alphanumeric characters
+            </mat-error>
+          </mat-form-field>
+        </div>
 
-              <mat-form-field class="w-100">
-                <input matInput
-                       placeholder="Licence Number from College"
-                       formControlName="licenseNumber"
-                       (blur)="onLicenceNumberBlur()">
-                <mat-error *ngIf="form.get('licenseNumber').hasError('required')">Required</mat-error>
-                <mat-error *ngIf="form.get('licenseNumber').hasError('alphanumeric')">
-                  Must contain only alphanumeric characters
-                </mat-error>
-              </mat-form-field>
+        <div class="col-sm-12 pl-5"
+             *ngIf="!condensed && prescriberIdMandatory > 1">
```

Comment:
Feels like there is a better way than using magic number "1", either defining the enum as a public property and refer in template, or define a method to return result for the ngIf

Reply:
I definitely agree